### PR TITLE
Project1

### PR DESCRIPTION
**Pull request by jeinygroove**
Can't deploy the web page to the live server because it requires @google.com, but that's how it looks in static: 
[screencapture-127-0-0-1-59004-src-main-webapp-index-html-2020-07-15-09_32_12.pdf](https://github.com/jeinygroove/step/files/4923290/screencapture-127-0-0-1-59004-src-main-webapp-index-html-2020-07-15-09_32_12.pdf)

### CODE REVIEW

**Review by amaurymedeiros — COMMENTED** [review]
On this first pass, can you please not include `font-awesome`? I'm not even sure if you're using it right now. From a first look, it seems you're only using `Lato` and you're obtaining that from Google fonts. Maybe you can include `font-awesome` in a future commit (we'd just need to check if the licensing is OK) if you need it later. Removing it from this pull request will make it much easier to be reviewed, as it draws the attention to the important files.

Another thing is it's better not to commit the `.DS_Store` files, since they're specific to macOS and don't need to be part of the project itself. One suggestion I can give is to add it to `.gitignore`, so it doesn't show up for you when you try to commit.

File: portfolio/src/main/webapp/index.html
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/index.html]: Oh OK, so I see you're using font-awesome here. So maybe you can leave the contacts session to be done in a future pull request, to make this one shorter...

File: portfolio/src/main/webapp/style.css
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/style.css]: From our [style guide](https://google.github.io/styleguide/htmlcssguide.html#Type_Selectors), it's not recommended to use type selectors. Maybe change `section.home` to `.home`? Same for other occurrences.

File: portfolio/src/main/webapp/style.css
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/style.css]: From the [style guide](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters), it's recommended to use hyphen (`-`) instead of underscore (`_`) to separate ID and class names. So in this case, `cat-description` would be a better naming. The same applies to other elements.

File: portfolio/src/main/webapp/script.js
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/script.js]: You probably don't need to keep this `console.log` here :)

File: portfolio/src/main/webapp/script.js
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/script.js]: Can you please add a short documentation to this function explaining what it does?

File: portfolio/src/main/webapp/script.js
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/script.js]: Can you please add a short documentation to this function explaining what it does?

File: portfolio/src/main/webapp/script.js
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/script.js]: That can probably be moved to a function with a meaningful name. For example, something like `setupScrollCallbacks`

File: portfolio/src/main/webapp/script.js
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/script.js:50]: It is a good practice to declare all functions together, so I would move this to the top of the file, close to makeScrollCallBack.

Then the rest of your code would also be together and easier to read, since there will be no functions declaration between statements.

File: portfolio/src/main/webapp/script.js
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/script.js]: type should be `number`, not `int`

File: portfolio/src/main/webapp/script.js
Comment (amaurymedeiros) [inline portfolio/src/main/webapp/script.js]: you don't need the square brackets (`[]`) here. So ` @param {int} sectionIndex Index number of the section.`

Same thing for return and for the other function

**Review by amaurymedeiros — COMMENTED** [review]
I have just minor comments, otherwise LGTM.  Waiting for Jerome's approval.

**Review by jgaillard85 — APPROVED** [review]

**Pull request merged**
